.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'blockprotocol/blockprotocol'
```

Comment:
What's this about? Isn't it always?

Reply:
I saw this `if` in one repo that uses `changesets` (not sure I remember which one). This is a protection against running the pipeline in a fork.